modules/ROOT/pages/ps-create-configure.adoc

```diff
@@ -0,0 +1,370 @@
+
+After the transit gateway attachment succeeds, you can attach your private space.
+
+For information about limitations when attaching a VPC to a transit gateway, 
+see 
+https://docs.aws.amazon.com/vpc/latest/tgw/tgw-vpc-attachments.html[Transit gateway attachments to a VPC^] in the AWS documentation.
+
+To attach a private space to the transit gateway:
+
+. In Anypoint Platform, on the *Transit Gateways* page, click *Attach Private Space*.
++
+The *Attach Private Space* button is disabled if no private spaces are available to attach or if the transit gateway addition failed. 
+. From the *Select Private Space* window, select the private space to attach.
++
+The window displays only the private spaces that you manage in the same region as the transit gateway.
+The window displays only the VPCs that are:
++
+--
+* In the same region as the transit gateway
+* Not already attached to the transit gateway
+// Can VPCs be added to multiple transit gateways?
+* In business groups for which you have CloudHub 2.0 Network Administrator permissions
+--
+. Click *Next*.
+. Sign in to your AWS corporate account.
+. In the *Accept VPC Attachment* window: 
++
+.. Click the *Transit Gateway Attachments* link. 
++
+The link opens the AWS RAM console to the page for accepting the transit gateway attachments in the region you specified.
+.. On the *Transit Gateway Attachments* page, select the attachment that shows *pending* in the *State* column.
++
+The attachment might take a few minutes to appear.
++
+To verify that the attachment is correct, select the transit gateway attachment ID and, in the *Details* tab, ensure that the *Resource owner account ID* is the MuleSoft AWS account ID from the *Add transit gateway* page.
+.. From the *Actions* menu, select *Accept*.
+. In Anypoint Platform, in the *Accept Private Space Attachment* window, click *Done*.
++
+The *Transit Gateways* page shows the progress for accepting the attachment from Anypoint Platform to AWS.
+
+When the VPC attachment succeeds, the *Transit Gateways* page displays the *Private space attached* message and the attachment state indicates that it's attached to AWS Transit Gateway.
+
+// If the VPC attachment state is `Rejected`, see xref:tgw-troubleshoot.adoc[Troubleshoot Transit Gateway Attachments].
+
+=== Configure Transit Gateway Routing
+
+CloudHub 2.0 supports static routing for transit gateways.
+
+Configure the network routes (subnets) that you want to be accessible through the transit gateway:
+
+* On Anypoint Platform, <<enable_outbound_traffic_from_ps,enable outbound traffic from the private space>> to an external destination.
+* On AWS, <<enable_inbound_traffic_on_aws,enable inbound traffic>> through your transit gateway. 
+
+[[enable_outbound_traffic_from_ps]]
+==== Enable Outbound Traffic from the Private Space
+
+After the *Private space attached* message appears, you can add routes so apps deployed to the private space can access the transit gateway.
```

Comment:
I know this is just copied from the CH TGW docs for now, but FYI the user supplies TGW routes at the beginning of the flow in CH2.0.

Reply:
Thanks. Fixed.